src/__tests__/__snapshots__/expandProperty-test.js.snap

```diff
@@ -545,8 +634,34 @@ Object {
 
 exports[`Expanding property values should expand text-decoration correctly 4`] = `
 Object {
-  "textDecorationColor": "underline",
-  "textDecorationLine": "navy",
+  "textDecorationColor": "navy",
+  "textDecorationLine": "underline",
   "textDecorationStyle": "dotted",
 }
```

Comment:
The previous expansion of `text-decoration` is wrong, the PR fixes that, and the snapshot is now updated.